### DISCUSSION TITLE
Why does the Frigate+ drop-down only have 11 object types?

### DISCUSSION THREAD

**Discussion by tve** [blakeblackshear/frigate#15676] [Ask A Question] [answered]
I'm trying to annotate a Frigate+ image (in "suggestions") and there's a waste_bin in the image but the drop-down to select the object type doesn't have waste_bin. What am I doing wrong?

![Screen-20241224-182811](https://github.com/user-attachments/assets/76d69958-faa0-4783-861a-6b3fd8723fee)

**Comment by NickM-27** [comment, marked as answer]
You edit the objects for each camera in the Frigate+ camera settings

**Reply by tve** [reply]
Thanks! I had forgotten that there's a separate config in frigate+. I wonder, however, why does it not import the config from what's live when an image is submitted? Or show a warning somewhere when the set of targets configured in f+ doesn't include all the ones configured in the running config?
